src/api/comments.ts

```diff
@@ -0,0 +1,37 @@
+export const BASE_URL = 'https://mate.academy/students-api/';
+
+export async function getComments(postId: number) {
+  const response = await fetch(`${BASE_URL}/comments?postId=${postId}`);
+
+  if (!response.ok) {
+    throw new Error(`${response.status} - ${response.statusText}`);
+  }
+
+  return response.json();
+}
+
+export async function delComment(id : number) {
+  const result = await fetch(`${BASE_URL}/comments/${id}`, { method: 'DELETE' });
+
+  return result;
+}
+
+export async function postComment(
+  name : string,
+  email : string,
+  body : string,
+  postId: number,
```

Comment:
Passing 4 arguments to a single function can lead to future bugs and mistakes, in this case it would be betterr and more readable to pass args as an object
```ts

interface PostComment  {
    name: string,
    email: string,
    body: string,
    postId: number,
}

export async function postComment({ name, email, body, postId } : PostComment)

```